action.yml

```diff
@@ -348,7 +348,7 @@ runs:
       shell: bash
     - uses: actions/cache@v4
       with:
-        key: cvmfs-apt-cache-${{ steps.lsb-release.outputs.id-release }}-${{ steps.lsb-release.outputs.arch }}-${{ hashFiles('action.yml') }}
+        key: cvmfs-apt-cache-${{ steps.lsb-release.outputs.id-release }}-${{ steps.lsb-release.outputs.arch }}
```

Comment:
In a repo using the existing Action version I see the cache key as
```
cvmfs-apt-cache-Ubuntu-Ubuntu 24.04.2 LTS-x86_64-
```
because the repo does not contain a file `action.yml` at the top level. This (I'm guessing) leads to undefined behaviour as the cache key is always missed